pom.xml

```diff
@@ -3,7 +3,7 @@
     <groupId>com.ksmpartners</groupId>
     <artifactId>domino-cli</artifactId>
     <packaging>jar</packaging>
-    <version>2.0.0</version>
+    <version>2.1.0-SNAPSHOT</version>
```

Comment:
I think this should be 3.0.0-SNAPSHOT, if only because support for at least one older version of Domino has been dropped (5.5.1, perhaps 5.6 too). Also because I plan to make more breaking changes to the CLI interface (more switches, less name-value pairs) and implementation (forcing all external data volumes to mount when running a job by default, which is currently not the case). Generally speaking:

- Major version changes will be forced by losing support for older versions of Domino; changing the CLI command structure in a breaking way; or changing the CLI behavior in a potentially breaking way (e.g. mounting all associated EDVs by default).
- Minor version changes will be forced by adding support for new versionMajor version changes will be forced by losing support for older versions of Domino; changing the CLI command structure in a breaking way; or changing the CLI behavior in a potentially breaking way.s of Domino (without dropping old versions off the back of the cart), or non-breaking CLI command structure changes (adding whole new commands, or new switches to existing commands that don't change the default behavior if they are not specified).
- Release changes will be for bug fixes only